knowledge-base/gantt-different-colors-for-tasks.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Set Different Colors for the Gantt Tasks
+description: How to set different colors for the Gantt tasks?
+type: how-to
+page_title: Set Different Colors for the Gantt Tasks
+slug: gantt-kb-different-colors-for-tasks
+position: 
+tags: 
+ticketid: 1536209
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Gantt for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I want to set different colors for the Gantt tasks. I store the colors in my data source. How to apply the colors from my data source to the rendered Gantt tasks?
+
+
+## Solution
+
+Follow these steps to apply the desired colors to the Gantt tasks:
+
+1. Use a [Task Template]({%slug gantt-task-template%}) to override the default task rendering.
+1. In the template, declare a wrapping element that will contain all the task details. Specify the desired task information inside.
+1. Add an attribute `style="background-color"` to the wrapper. Set the color from the template `context` or any other desired color.
```

Comment:
Please, double-check the second sentence here as it sounds weird. It might be correct but it's a bit confusing.

Reply:
> Please, double-check the second sentence here as it sounds weird. It might be correct but it's a bit confusing.

That's alright. It implies color can be set both from the context model passed to the template (e.g. predefined based on the task), or an abitrary color alltogether.